src/Raven.Server/Config/RavenConfiguration.cs

```diff
@@ -249,9 +260,10 @@ public void PostInit()
             }
             catch (Exception e)
             {
-                if (_logger.IsOperationsEnabled)
-                    _logger.OperationsWithWait("Invalid security configuration. Stopping RavenDB server startup", e)
-                        .Wait(UnhandledExceptions.TimeToWaitForLog); // using async version to wait and ensure that we'll wait for the log to be written to disk since we're gonna break the service startup
+                if (_logger.IsFatalEnabled)
+                    _logger.Fatal("Invalid security configuration. Stopping RavenDB server startup", e);
```

Comment:
Fatal won't throw if NLog isnt yet initialized right. Maybe we should somehow fallback to Console.WriteLine?